docs/examples.md

```diff
@@ -514,6 +513,25 @@ gcloud functions deploy local-nuget \
   --entry-point Google.Cloud.Functions.Examples.LocalNuGetPackageFunction.Function
 ```
 
+## Typed GoogleChatBot
+
+The [GoogleChatbot](../examples/Google.Cloud.Functions.Examples.GoogleChatBot)
+sample implements the [Google Chat App
+API](https://developers.google.com/chat/how-tos/apps-develop) to create a simple
+bot that replies to messages with a copy of your Google Chat profile image. See
+[google chat quickstart](https://developers.google.com/chat/quickstart/gcf-app)
```

Comment:
google chat quickstart => Google Chat Quickstart

(I haven't added this as a suggestion, as that causes CLAbot issues...)